pkg/pb/synthetic_monitoring/checks_extra_test.go

```diff
@@ -801,6 +804,119 @@ func TestHttpSettingsValidate(t *testing.T) {
 	}
 }
 
+func TestCompressionAlgorithmMarshal(t *testing.T) {
+	type testStruct struct {
+		Compression CompressionAlgorithm `json:"compression,omitempty"`
+	}
+
+	testcases := map[string]struct {
+		unserialized testStruct
+		serialized   []byte
+	}{
+		"none": {
+			unserialized: testStruct{
+				Compression: CompressionAlgorithm_none,
+			},
+			serialized: []byte(`{}`),
```

Comment:
Just to be sure, this should evaluate to no decrompression if there is no `compression` key _or_ `compression: none` _or_ `compression: ""`, right?

Reply:
@rdubrock that's right.